data/persons.txt

```diff
@@ -944,23 +942,24 @@ person "Prototype B3-CC4"
 		word
 			">"
 	ship "Shooting Star" "Prototype B3-CC4"
-		"never disabled"
 
 ship "Shooting Star"
+	plural "Shootings Star"
+	noun "genocidal maniac"
 	sprite "ship/shooting star/shooting star"
 		"frame time" 3
-	"never disabled"
 	attributes
 		category "Heavy Warship"
 		"hull" 31500
-		"required crew" 1
-		"bunks" 2
+		"automaton" 200
+		"cost" -49500000
```

Comment:
I could just disown it with no cost to me. I'd just ditch the negative cost.

Reply:
Eh, I'd rather just keep it, I bet there will be someone who didn't think of that. Me, for instance